.storybook/storybook.css

```
@@ -0,0 +1,295 @@
+  border: 1px solid #e9ecef;
+}
+
+/* Blockquote styling */
+.sbdocs-content blockquote {
+  border-left: 4px solid #0066cc;
+  padding-left: 1rem;
+  margin: 1.5rem 0;
+  color: #666;
+  font-style: italic;
+  background: #f8f9fa;
+  padding: 1rem;
```

Biome (2.3.13)

[error] Unexpected shorthand property padding after padding-left



(lint/suspicious/noShorthandPropertyOverrides)